src/display/annotation_layer.js

```diff
@@ -96,14 +97,17 @@ AnnotationElementFactory.prototype =
  * @alias AnnotationElement
  */
 var AnnotationElement = (function AnnotationElementClosure() {
-  function AnnotationElement(parameters) {
+  function AnnotationElement(parameters, isRenderable) {
+    this.isRenderable = isRenderable;
```

Comment:
This can be even easier, like in Snuffleupagus' solution: just return `null` there if `isRenderable` is `false`. That way we can remove `isRenderable` entirely.